### PR TITLE
DAOS-10161 test: Measure dmg pool list duration and print in pool/cre…

### PR DESCRIPTION
**Pull request by shimizukko**
…ate_capacity.py

Some customer reported that dmg pool list takes a long time. In pool/create_capacity.py, 200 pools are created and listed, so measure the time it takes to list and print it as a reference.

Skip-unit-tests: true
Test-tag: create_performance
Signed-off-by: Makito Kano <makito.kano@intel.com>

### CODE REVIEW

**Review by dinghwah — APPROVED** [review]
lgtm

File: src/tests/ftest/pool/create_capacity.py [hunk 1]
```diff
@@ -60,7 +60,12 @@ def test_create_pool_quantity(self):
         self.dmg.timeout = 360
 
         # Verify all the pools exists after the restart
+        pool_list_start = float(time.time())
         pool_uuids = self.get_dmg_command().get_pool_list_uuids(no_query=True)
```
[the remark this answers is not in the document]
Comment (shimizukko) [reply src/tests/ftest/pool/create_capacity.py]: Thanks for pointing out. You're right, we shouldn't measure time with `no_query=True`. I added a block to call pool list and measure the time there instead.

**Review by dinghwah — APPROVED** [review]
If there is any pass/fail check for the pool_list_duration?
Or just a reference?

**Comment by shimizukko** [comment]
@daltonbohning Thanks for the comment. I realized that this is a PR test, so adding 200 sec is probably not a good idea. Also, if you guys are planning to cover this type of tests, then I think it's better not to add this test here.

@dinghwah No, this is just for reference to see whether dmg pool list is taking unreasonable amount of time.

**Pull request closed without merging**